src/runtime/virtcontainers/kata_agent.go

```diff
@@ -2014,15 +2029,15 @@ func (k *kataAgent) installReqFunc(c *kataclient.AgentClient) {
 	}
 }
 
-func (k *kataAgent) getReqContext(reqName string) (ctx context.Context, cancel context.CancelFunc) {
-	ctx = context.Background()
+func (k *kataAgent) getReqContext(ctx1 context.Context, reqName string) (ctx context.Context, cancel context.CancelFunc) {
```

Comment:
Please, use "ctx" and "newCtx" instead of "ctx1" and "ctx"

Reply:
@fidencio have updated as your advice.